src/helper/tokenToData.js

```diff
@@ -0,0 +1,16 @@
+import jwt from 'jsonwebtoken'
+import isUserExist from '../services/findUser';
+export async function getDataFromToken(req, res, next){
+    if(req.headers && req.headers.authorization){
+        const authorization = req.headers.authorization
+        var decoded ='';
+        try {
+            decoded = jwt.verify(authorization, process.env.TOKEN_SECRET);
```

Comment:
should we use the pre created reusable function for this

Reply:
I used it as suggested, thanks